tools/frr-reload.py

```diff
@@ -779,13 +779,14 @@ def ignore_delete_re_add_lines(lines_to_add, lines_to_del):
                     bfd_search_string =  bfd_nbr + r' bfd (\S+) (\S+) (\S+)'
 
                     for (ctx_keys, add_line) in lines_to_add:
-                        re_add_nbr_bfd_timers = re.search(bfd_search_string, add_line)
+                        if ctx_keys[0].startswith('router bgp'):
```

Comment:
isn't this checked already in this method above?

Reply:
No because at this point we are spinning thru whatever is in lines_to_add, which includes anything being added.